applications/luci-app-cloudflared/root/usr/share/luci/menu.d/luci-app-cloudflared.json

```diff
@@ -0,0 +1,29 @@
+{
+	"admin/vpn/cloudflared": {
```

Comment:
I'm not sure if it should be placed in `VPN` menu, as it doesn't work like a "typical VPN".

Reply:
More apt might be one of ZTN > ZTS > ZTA. For networking, security and zero trust architecture respectively. Not many products out there though. 

VPN seems suitable when compared to other general alternatives.